docs/source/conf.py

```diff
@@ -18,14 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "Amsterdam DSO API"
-copyright = (
-    "2020, Team Dataservices. "
-    "DataPunt, afdeling Onderzoek, Informatie en Statistiek Gemeente Amsterdam."
-)
-author = (
-    "Team Dataservices. "
-    "DataPunt, afdeling Onderzoek, Informatie en Statistiek Gemeente Amsterdam."
-)
+copyright = "2021, Team Data Diensten, Gemeente Amsterdam."
+author = "Team Data Diensten, Gemeente Amsterdam."
```

Comment:
Is het niet Datadiensten?

Reply:
Het stond met een spatie in azure, dus heb ik dat zo overgenomen.